src/linear_algebra/matrix/adjugate.lean

```diff
@@ -209,6 +222,20 @@ begin
     exact h ((symm_apply_eq σ).mp h') }
 end
 
+lemma adjugate_submatrix_equiv_self (e : n ≃ m) (A : matrix m m α) :
+  adjugate (A.submatrix e e) = (adjugate A).submatrix e e :=
+begin
+  ext i j,
+  rw [adjugate_apply, submatrix_apply, adjugate_apply, ← det_submatrix_equiv_self e,
+    update_row_submatrix_equiv],
+  congr,
+  exact function.update_comp_equiv _ e.symm _ _,
+end
+
+lemma adjugate_reindex (e : m ≃ n) (A : matrix m m α) :
+  adjugate (reindex e e A) = reindex e e (adjugate A) :=
```

Comment:
and this one too?

Reply:
`reindex` isn't simp-normal, so not this one.